frontend/src/container/LogsPanelTable/LogsPanelComponent.styles.scss

```diff
@@ -63,6 +63,8 @@
 		height: 40px;
 		justify-content: end;
 		padding: 0 8px;
+		margin-top: 12px;
```

Comment:
combine them into one `margin` property ?

Reply:
+1

Reply:
Adding changed under this PR - https://github.com/SigNoz/signoz/pull/6056